script/lib/lint-java-script-paths.js

```diff
@@ -6,29 +6,46 @@ const path = require('path')
 
 const CONFIG = require('../config')
 
-module.exports = function () {
+module.exports = async function () {
   const globPathsToLint = [
     path.join(CONFIG.repositoryRootPath, 'exports', '**', '*.js'),
     path.join(CONFIG.repositoryRootPath, 'packages', '**', '*.js'),
     path.join(CONFIG.repositoryRootPath, 'script', '**', '*.js'),
+    path.join(CONFIG.repositoryRootPath, 'spec', '**', '*.js'),
     path.join(CONFIG.repositoryRootPath, 'src', '**', '*.js'),
     path.join(CONFIG.repositoryRootPath, 'static', '*.js')
   ]
-  return expandGlobPaths(globPathsToLint).then((paths) => {
-    return new Promise((resolve, reject) => {
-      standard.lintFiles(paths, (error, lintOutput) => {
-        if (error) {
-          reject(error)
-        } else {
-          const errors = []
-          for (let result of lintOutput.results) {
-            for (let message of result.messages) {
-              errors.push({path: result.filePath, lineNumber: message.line, message: message.message, rule: message.ruleId})
-            }
+  const globPathsToIgnore = [
+    path.join(CONFIG.repositoryRootPath, 'spec', 'fixtures', '**', '*.js')
+  ]
+
+  const [includePaths, excludePaths] = await Promise.all([
+    expandGlobPaths(globPathsToLint),
+    expandGlobPaths(globPathsToIgnore)
+  ])
+
+  const paths = includePaths.filter(
+    myPath => excludePaths.indexOf(myPath) === -1
```

Comment:
```suggestion
    myPath => !excludePaths.includes(myPath)
```
[Array.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) is almost always clearer for showing the intent of the code.